stdlib/ballerina-websub/src/main/ballerina/websub/hub_caller_actions.bal

```diff
@@ -201,10 +201,18 @@ public function CallerActions::publishUpdate(string topic, json payload, string?
 
     var response = httpClientEndpoint->post(untaint ("?" + queryParams), request = request);
     match (response) {
-        http:Response => return;
-        error httpConnectorError => { error webSubError = {
-            message:"Notification failed for topic [" + topic + "]", cause:httpConnectorError};
-        return webSubError;
+        http:Response response => {
+            string respStatusCode = <string>response.statusCode;
+            if (!respStatusCode.hasPrefix("2")) {
```

Comment:
Wouldn't it be nice if we check whether the status code is between successful range?

Reply:
Fixed with fd6e70f, introducing a new function.

Reply:
Fixed with fd6e70f, introducing a new function.